content/02.delete-me.md

```diff
@@ -1,15 +1,165 @@
+
+<!-- an arbitrary comment. not visible in output. -->
+
+Horizontal rule:
+
+---
+
+A long paragraph of text.
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+Another long paragraph.
+Donec massa sapien faucibus et molestie ac feugiat sed lectus.
+Est sit amet facilisis magna etiam.
+Ornare arcu dui vivamus arcu felis bibendum.
+Magna fermentum iaculis eu non diam.
+Vehicula ipsum a arcu cursus vitae.
+Ante metus dictum at tempor commodo ullamcorper a.
+Vitae tortor condimentum lacinia quis vel eros.
+Ut venenatis tellus in metus vulputate eu scelerisque felis imperdiet.
+Facilisis sed odio morbi quis.
+Enim ut sem viverra aliquet eget sit amet tellus.
+
+*Italic* _text_
+
+**Bold** __text__
+
+Combined *italics and __bold__*
+
+~~Strikethrough~~
+
+Line break without starting a new paragraph by putting  
+two spaces at end of line.
+
+Putting each sentence on its own line has numerous benefits with regard to [editing](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line) and [version control](https://rhodesmill.org/brandon/2012/one-sentence-per-line/).
+
+1. Ordered list item
+2. Ordered list item
+    a. Sub-item
+    b. Sub-item
+        i. Sub-sub-item
+3. Ordered list item
+    a. Sub-item
+
+- List item
+- List item
+- List item
+
+superscript<sup>2</sup>
+
+subscript<sub>2n+1</sub>
+
+unicode superscript™
+
+unicode subscriptₓ
+
+**Links and Citations**
+
+Bare URL link: <https://manubot.org>
+
+[Link with text](https://www.manubot.org)
+
+[Link with hover text](https://www.manubot.org "Manubot Homepage")
+
+[Link styled as a button](https://www.manubot.org "Manubot Homepage"){.button}
+
+[Link by reference][manubot homepage]
+
+[Manubot Homepage]: https://www.manubot.org
+
+Citation via DOI: [@doi:10.15363/thinklab.4]
+
+Citation by tag: [@tag:deep-review]
+
+Manual URL citation: [@url:https://greenelab.github.io/meta-review]
+
+**Figures and Tables**
+
+Figure @fig:square-image
+
+Figure @fig:wide-image
+
+Figure @fig:tall-image
+
+Figure @fig:vector-image
+
+Table @tbl:bowling-scores
+
+**Quotes and Code**
+
+> Quoted text
+
+> Quoted block of text  
+>
+> Two roads diverged in a wood, and I—  
+> I took the one less traveled by,  
+> And that has made all the difference.  
+
+Code `in the middle` of normal text, aka `inline code`
+
+```c++
+// code block with C++ syntax highlighting
+int main() {
+    // a string too long to fit within page
+    cout << "It was the best of times, it was the worst of times, it was the age of wisdom, it was the age of foolishness";
+    return 0;
+}
+```
```

Comment:
Should we change this to something more project relevant? I do like the poetry, but another option would be:


```python
from manubot.cite.doi import expand_short_doi

def test_expand_short_doi():
    doi = expand_short_doi('10/b6vnmd')
    assert doi == "10.1016/s0933-3657(96)00367-3"
```

Reply:
The reason for that poetry was to provide a sentence that was really long, forcing the code box to scroll, to make sure we support that properly (pandoc's generic CSS does not handle it correctly). If you can think of some project relevant code that is necessarily really long horizontally, let me know.

Reply:
Here are the longest crossref DOIs in existence (after some filters):

```python
['10.25313/2524-2695-2018-3-vliyanie-enhansera-copia-i-insulyatora-gypsy-na-sintez-ernk-modifikatsii-hromatina-i-svyazyvanie-insulyatornyh-belkov-vtransfetsirovannyh-geneticheskih-konstruktsiyah',
 '10.25313/2524-2695-2018-5-doslidzhennya-metodiv-personalizovanogo-obslugovuvannya-koristuvachiv-web-resursiv-z-metoyu-pidboru-najvigidnishoyi-propozitsiyi',
 '10.25313/2524-2695-2018-4-osobennosti-funktsionirovaniya-finansovo-ekonomicheskogo-mehanizma-gosudarstvennogo-regulirovaniya-ustojchivogo-razvitiya',
 '10.25313/2524-2695-2018-5-strategicheskoe-planirovanie-kak-instrument-obespecheniya-finansovoj-ustojchivosti-predpriyatiya',
 '10.1615/atoz.c.comitato_nazionale_per_la_ricerca_e_per_lo_sviluppo_dellenergia_nucleare_e_delle_energie_alternative_enea']
```

I'm thinking we can replace the previous example with the longest DOI.

```python
from manubot.cite.doi import expand_short_doi

def test_expand_short_doi():
    doi = expand_short_doi('10/c3bp')
    assert doi == "10.25313/2524-2695-2018-3-vliyanie-enhansera-copia-i-insulyatora-gypsy-na-sintez-ernk-modifikatsii-hromatina-i-svyazyvanie-insulyatornyh-belkov-vtransfetsirovannyh-geneticheskih-konstruktsiyah"
```

It's a cool demonstration of shortDOIs that the target manubot audience may find somewhat comical